js/shared.js

```diff
@@ -147,6 +147,11 @@ const signInFlowRender = (signInEmail) => {
 };
 
 export const sendEmailLink = () => {
+    //Check for language storage
+    let preferredLanguage = window.localStorage.getItem('preferredLanguage');
```

Comment:
sorry, I was confused from what you sent... you should be able to reference `getSelectedLanguage()` within `shared.js`

Reply:
Solved